### PR TITLE
[destination-typesense] Connect to typesense with HA #42866

### PR DESCRIPTION
**Pull request by edimarlnx**
## What

This change allows for the specification of more than one host on the host field, separated by commas, and transforms it into multiple connection nodes.

https://github.com/airbytehq/airbyte/issues/42866

## User Impact
I believe it does not cause user impacts.

## Can this PR be safely reverted and rolled back?
- [x] YES 💚
- [ ] NO ❌

### CODE REVIEW

File: airbyte-integrations/connectors/destination-typesense/destination_typesense/destination.py [hunk 1]
```diff
@@ -14,11 +14,15 @@
...
 def get_client(config: Mapping[str, Any]) -> Client:
-    node = {"host": config.get("host"), "port": config.get("port") or "8108", "protocol": config.get("protocol") or "https"}
+    hosts = config.get("host").split(",")
     path = config.get("path")
-    if path:
-        node["path"] = path
-    client = Client({"api_key": config.get("api_key"), "nodes": [node], "connection_timeout_seconds": 3600})
+    nodes = []
+    for host in hosts:
+        node = {"host": host, "port": config.get("port") or "8108", "protocol": config.get("protocol") or "https"}
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/destination-typesense/destination_typesense/destination.py:21]: You're considering if they user have different servers/host they're using the same port and protocol. Is this correct?

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
Left a comment! Thanks for contributing <3

File: airbyte-integrations/connectors/destination-typesense/destination_typesense/destination.py (hunk 1 above)
Comment (edimarlnx) [reply airbyte-integrations/connectors/destination-typesense/destination_typesense/destination.py:21]: Correct. This is the way I found to allow connection without impacting what exists today.

I think this approach will meet most cases. You could keep the same port and protocol unless you run more than one instance on the same machine.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
@edimarlnx can you bump the metadata, pyproject and the doc changelog entry?

**Comment by edimarlnx** [comment]
> @edimarlnx can you bump the metadata, pyproject and the doc changelog entry?

Sure. 
Can you review the changes?

**Review by marcosmarxm — APPROVED** [review]
LGTM

**Pull request merged**